### PR TITLE
gnumeric: Add package

### PR DESCRIPTION
**Pull request by stahta01**
Requires [PR 8997](https://github.com/msys2/MINGW-packages/pull/8997)  to build.

Edit: Added goffice package to this branch to try avoiding 8997

Tim S.

### CODE REVIEW

**Comment by Biswa96** [comment]
You could add this package in that PR. CI will automatically build goffice before gnumeric.

**Comment by stahta01** [comment]
Why CI is broken in this repo and has been for over a month.

**Comment by jeremyd2019** [comment]
> Why CI is broken in this repo and has been for over a month.

Because nobody really uses it, so when I fixed the other CIs I missed this one?

**Comment by jeremyd2019** [comment]
OK, CI updated here to match MINGW-packages, but you'd still be better off adding this to your existing pull request over there so CI could build both packages.

File: mingw-w64-goffice/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Contributor: Tim Stahlhut <stahta01@gmail.com>
+
+_realname=goffice
+_sourcedir=${_realname}-git
+pkgbase=mingw-w64-${_realname}
+pkgname=(
+  "${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+)
```
Comment (jeremyd2019) [inline mingw-w64-goffice/PKGBUILD:8]:
```suggestion
)
provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```

**Comment by stahta01** [comment]
Decide I need to work on this more before doing an PR

**Comment by Biswa96** [comment]
You can test the package in your forked repository. May GitHub Action be with you!

**Pull request closed without merging**